index.html

```diff
@@ -201,8 +202,43 @@ <h4 class="connect">LET'S CONNECT</h4>
               Skills <img src="./images/arrow-forward.png" alt="" />
             </li>
           </ul>
+          </div>
         </section>
       </section>
+
+      <section class="contact">
```

Comment:
- [ ] Kindly check on your design, remove the fixed width from your page design, it's making the design break and be static instead of responsive.